clients/client-java/src/main/java/com/datastrato/gravitino/client/RelationalTable.java

```diff
@@ -57,7 +57,7 @@ public static RelationalTable from(
   /**
    * Creates a new RelationalTable.
    *
-   * @param namespace The namespace of the table.
+   * @param namespace The full namespace of the table, including metalake, catalog and schema
```

Comment:
I found that `RelationTable` as a public constructor, we should make it package private, also it has a method `namespace()`, I'm not sure if it is needed, if we want to expose it, then the namespace should be the relative namespace, right?

Reply:
I had though a relative namespace is okay for `RelationalTable`, but I noticed there are several methods to operation the partitions, like `listPartitions()`, `addPartition()`, which will send another HTTP request to the server side, in which it need a full context. So, base on this, the `RelationalTable` need hold its full namespace information. Otherwise, the interface `SupportsPartitions` won't be implemented easily on the `RelationalTable` level.

Reply:
Yeah, I know that `RelationalTable` needs a full namespace to support partition operation, what I mainly mentioned is the method `namespace()`, which is user facing (public), either we can drop this method if no one uses it, or we need to thinking of to return a relative namespace for user.

Reply:
Done, droped the "namespace()" method.